samples/cpp/cloning_gui.cpp

```diff
@@ -453,7 +453,7 @@ int main()
 
     for(;;)
     {
-        char key = (char) waitKey(0);
+        int key = waitKey(0) & 0xFF;
 
         if(key == 'd' && flag3 == 0)
```

Comment:
Hi @apavlenko, I guess I am missing what you are trying to say here. Could you explain where you would like in INT cast to ensure that we have no loss of values above 127? Or am I completely missing your point?